packages/service-library/src/servicelib/json_serialization.py

```diff
@@ -1,16 +1,28 @@
 import json
+from collections.abc import Callable
 from typing import Any
 
-from models_library.utils.fastapi_encoders import jsonable_encoder
+import orjson
 from pydantic.json import pydantic_encoder
 
 
 def json_dumps(obj: Any, **kwargs):
-    """json.dumps with rich encoder.
+    """json.dumps including a rich encoder (e.g. supports UUID objs)
     A big applause for pydantic authors here!!!
     """
     return json.dumps(obj, default=pydantic_encoder, **kwargs)
 
 
-def safe_json_dumps(obj: Any, **kwargs):
-    return json_dumps(jsonable_encoder(obj), **kwargs)
+class OrJsonAdapter:
```

Comment:
Cool, did you check that OrJson is actually solves the problem with the slower socketio communication?

Reply:
we will see. `orjson` promises speed.